.pre-commit-config.yaml

```diff
@@ -114,7 +114,8 @@ repos:
     hooks:
       - id: codespell
         additional_dependencies: ["tomli"]
-        args: ["-L", "lama", "--ignore-regex", ".{1024}|.*codespell-ignore.*"]
+        args:
+          ["-L", "lama,witha", "--ignore-regex", ".{1024}|.*codespell-ignore.*"]
```

Comment:
(Note, not for now) if this list grows, we should probably use a words file instead